README.md

```diff
@@ -26,6 +26,25 @@ This quick overview will help you get started developing in the Babylon React Na
 
 If you are interested in making contributions, be sure to also review [CONTRIBUTING.md](CONTRIBUTING.md).
 
+### **Chosing a react-native version**
+
+Build script and Package building scripts expect to use a folder named `Apps/Playground/Playground`. Create a symlink to a react-native version in `Apps/Playground`.
+For a react-native 0.64 version:
+
+on MacOS/Linux:
+```
+cd Apps/Playground
+ln -s 0.64 Playground
+```
+
+on Windows:
+```
+cd Apps/Playground
+mklink /J Playground 0.64
+```
```

Comment:
Maybe for another PR, but it seems like we could have a node.js script in the root that could do this for you so that regardless of platform, you would just run a node command to create the sym link).

Reply:
There is a way to push symlinks in Git. So a default symlink to 0.64 would make the current readme build steps to be the same. user would change the symlink (or the CI) when needing a different version. But git symlinks on Windows are not managed the same way and you need special configuration to get them supported. Not ideal.

Creating a symlink with a default to 0.64 seems great but when shall it be done? When running npm install you already have to be in the correct folder. Or run npm install in Apps/Playground, check for an env var and create symlink at that moment and continue installation in the symlinked folder?

Reply:
I don't think it necessarily needs to be automatic, I just think it'd be easier to use if the command was the same across Windows/Mac/Linux. Something like `npm run selectRNVersion --version 0.64`. I guess to have something like that, it would probably need to be an npm script in `Apps/Playground`, but then in that case you might as well have a default version as part of a npm post install script in that dir. Then it would be as you mentioned above, where you would `npm install` in the Apps/Playground directory and it could do the symlink and run `npm install` in the Apps/Playground/Playground dir to continue repo setup.

Reply:
I like the idea.